bridge_adaptivity/bridge_lti/templates/bridge_lti/content-source.html

```diff
@@ -17,14 +18,8 @@
   <button type="submit">Launch the tool</button>
 </form>
 
-<script type="text/javascript">
-    (function (d) {
-        var element = d.getElementById("ltiLaunchForm");
-        if (element) {
-            element.submit();
-        }
-    }(document));
-</script>
+{% bootstrap_javascript jquery=True %}
```

Comment:
jquery already included in settings, so this string looks odd, or I have missed something. What is really needed in configure jquery cdn url, by default v1.1 is used I would use v3.2.1

Reply:
get `jQuery is not defined at bridge-lti.js:11` if cut it out...